lib/Transforms/DataCopy.cpp

```diff
@@ -34,46 +40,50 @@
 #include "llvm/Support/Debug.h"
 #include <algorithm>
 
-#define DEBUG_TYPE "affine-dma-generate"
+#define DEBUG_TYPE "affine-data-copy"
 
 using namespace mlir;
 using llvm::SmallMapVector;
 
 static llvm::cl::OptionCategory clOptionsCategory(DEBUG_TYPE " options");
 
 static llvm::cl::opt<unsigned long long> clFastMemoryCapacity(
-    "dma-fast-mem-capacity",
+    "copy-fast-mem-capacity",
     llvm::cl::desc(
         "Set fast memory space capacity in KiB (default: unlimited)"),
     llvm::cl::cat(clOptionsCategory));
 
+static llvm::cl::opt<bool>
+    clDma("dma", llvm::cl::desc("Generate DMA instead of point-wise copy"),
```

Comment:
All of these cl options should start with the pass arg, otherwise they don't make sense.
https://github.com/tensorflow/mlir/blob/master/g3doc/DeveloperGuide.md#pass-name-and-other-command-line-options

Reply:
Done.